### PR TITLE
fix: Add ektaarora16

### LINKED ISSUE #749: ektaarora16
Ektaa

### PR DESCRIPTION
**Pull request by ektaarora16**
resolved #749

### CODE REVIEW

**Review by CodingSpecies — APPROVED** [review]
LGTM, thank you for contributing @ektaarora16 !

**Review by AaquibAli — APPROVED** [review]

File: data.json [hunk 1]
```diff
@@ -1385,4 +1385,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/73016212/135509959-87f8b8e4-9fab-4247-bae4-d965fca5ddbc.png",
     "issueNumber": 742
   }
```
Comment (schmelto) [inline data.json:1387]:
```suggestion
  },
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by KukretiShubham — APPROVED** [review]
after changes by Tom.
LGTM

**Comment by ektaarora16** [comment]
changes done. Please revive it

**Comment by ektaarora16** [comment]
Please review @schmelto

**Review by KukretiShubham — APPROVED** [review]
LGTM 🚀

**Comment by KukretiShubham** [comment]
Hey, there are merge conflict. Hope maintainers will resolve. And @ektaarora16 you can also avoid merge conflict by fetching the upstream on your forked repo.

File: data.json [hunk 2]
```diff
@@ -1384,5 +1384,11 @@
     "githubUsername": "DarshGupta1910",
     "imageUrl": "https://user-images.githubusercontent.com/73016212/135509959-87f8b8e4-9fab-4247-bae4-d965fca5ddbc.png",
     "issueNumber": 742
+  },
+  {
+    "name": "Ekta Arora",
+    "githubUsername": "ektaarora16",
+    "imageUrl": "https://user-images.githubusercontent.com/90129737/135709011-e377fce0-d0b9-46d5-8eff-f2edec70e8fa.png",
```
Comment (Panquesito7) [inline data.json:1391]: I would recommend more of an image like this.

image

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/135740246-b0be6db0-160a-43d8-819d-0ad3c190619c.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the [**merge conflicts**](https://github.com/EddieHubCommunity/awesome-github-profiles/pull/750/conflicts). If you need any help, don't hesitate to ask for help here or in our [**Discord server**](http://discord.eddiehub.org). 🙂

**Comment by ektaarora16** [comment]
Resolved conflict. Please revive it.

**Review by KukretiShubham — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**